spec/factories/users.rb

```diff
@@ -58,4 +59,14 @@
     biography { '<div id="divInjectedElement"></div>' }
   end
 
+  factory :organizer, parent: :user do
+    transient do
+      resource { nil }
```

Comment:
[nitpick] what about having a resource by default?

Reply:
Done it in last commit.